### ISSUE TITLE
UI terminology / translations: replace the term "paste" with the more generic "document"

### ISSUE THREAD

**Issue by elrido** [PrivateBin/PrivateBin#397]
We recently had merge requests for the [German](https://github.com/PrivateBin/PrivateBin/pull/382) and [Spanish](https://github.com/PrivateBin/PrivateBin/pull/394) translations that replaced the term "paste" with the more generic "text".

Our project isn't called "PasteBin", so I agree that there is no need for us to use the term "paste" for branding reasons and it isn't a word in English or other languages anyway. I do feel mildly uncomfortable with the use of "text", since admins can optionally enable the upload of attachments along side the (pasted) text and with markdown and relaxed  [CSP headers](https://github.com/PrivateBin/PrivateBin/wiki/Configuration#cspheader) external resources can be embedded as well.

How about using "document" instead of "paste"?

It's very generic and it works well for any use case. Same as in a word processor or presentation document you can include images and other embedded media in a PrivateBin "document".

**Comment by rugk** [comment]
Yeah, why not? Anyone, who knows some counter arguments, why we should not do so?

**Comment by BrookeDot** [comment]
"Content" may also work here.

**Comment by rugk** [comment]
For context, I guess we should frame [the text](https://github.com/PrivateBin/PrivateBin/blob/879a2a92550b1df3d6b6be62644aea51d4bd30a5/i18n/de.json) in the whole sentence, so one may see how it "sounds" if it is included like that. Some examples:

### currently "paste"
* > Paste does not exist, has expired or has been deleted.
* > Paste is limited to %s of encrypted data.
* > Error saving paste. Sorry.
* > Invalid paste ID.
* > Paste is not of burn-after-reading type.
* > Wrong deletion token. Paste was not deleted.
* > Please enter the password for this paste:
* > Sending paste…
* > The cloned file '%s' was attached to this paste.
* > +++ no paste text +++

### text
* > Text does not exist, has expired or has been deleted.
* > Text is limited to %s of encrypted data.
* > Error saving text. Sorry.
* > Invalid text ID.
* > Text is not of burn-after-reading type.
* > Wrong deletion token. Text was not deleted.
* > Please enter the password for this text:
* > Sending text…
* > The cloned file '%s' was attached to this text. [??]
* > +++ no text text +++ [???]

The existing translation work around the last two problems by omitting the duplication (saying "The cloned file '%s' was attached.") or keeping "paste" literally ("+++ kein Paste-Text +++" in German e.g.).

### "document"
* > Document does not exist, has expired or has been deleted.
* > Document is limited to %s of encrypted data.
* > Error saving document. Sorry.
* > Invalid document ID.
* > Document is not of burn-after-reading type.
* > Wrong deletion token. Document was not deleted.
* > Please enter the password for this document:
* > Sending document…
* > The cloned file '%s' was attached to this document.
* > +++ no document text +++ [??]

The last one sounds awkward, too. (could be changed to "no text in document")
Everything else sounds quite reaasonable.

### "content"
* > Content does not exist, has expired or has been deleted.
* > Content is limited to %s of encrypted data.
* > Error saving content. Sorry.
* > Invalid content ID.
* > Content is not of burn-after-reading type.
* > Wrong deletion token. Content was not deleted.
* > Please enter the password for this content:
* > Sending content…
* > The cloned file '%s' was attached to this content.
* > +++ no content text +++ [???]

IMHO a little too documentary and too "technology-like". It's generic, that is okay, but sounds akward sometimes.

**Comment by elrido** [comment]
Ok, so "Paste" and "Text" are confusing as soon as one attaches a document, image or uses Markdown. "Document" sounds a bit boring and can be a bit confusing with images. "Content" is still a bit technical (like "Paste"), but I think it is more generic and better known to non-technical users.

So would "Content" be OK with you, too, @rugk?

**Comment by rugk** [comment]
I'd tend to "Document".

**Comment by elrido** [comment]
Ok, let's poll some folks - feel free to share this link in social media with other PrivateBin users:
https://www.systemli.org/poll/#/poll/WO1PEAFQ5K/participation?encryptionKey=hEbthZLaLJupBHwsAKJoUIVLMWzSF6ozcHSDb2yl

**Comment by Cellophile** [comment]
And what about "message" ? All things we put in privatebin are shared with others, so we can consider it like a message !
